UIKit_DZ_Swift2.0/UIKit_DZ_Swift2.0/PizzaData.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  PizzaData.swift
+//  UIKit_DZ_Swift2.0
+//
+//  Created by Ernest Avagovich on 28.09.2022.
+//
+
+import Foundation
+
+/// Описывает название пиццы и картинку
+///  name - Название пиццы
+///  imageName - Название картинки пиццы
+struct Pizza {
+    let name: String?
+    let imageName: String?
+}
+
+/// Описыввает названия ингредиентов
+struct PizzaIngredients {
```

Comment:
```suggestion
struct PizzaIngredient {
```
тут множественное не надо так как модель описывает 1 элемент